src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ImportRemoteInstanceController.cs

```diff
@@ -43,9 +43,14 @@ public async Task<IActionResult> Import(ImportViewModel model)
 
             var remoteClient = remoteClientList.RemoteClients.FirstOrDefault(x => x.ClientName == model.ClientName);
 
+            if (remoteClient == null)
+            {
+                return StatusCode((int)HttpStatusCode.BadRequest, "The remote client was not provided");
```

Comment:
Just for info the following does the same, yes better to not do it here for consistency with other lines of code below, but maybe a good canditate for a separate PR ;)

                return BadRequest("The remote client was not provided");

Reply:
I will try to check the entire code to see if we have such thing .. thanks

Reply:
#12344